V3-Upgrade-Guide.md

```diff
@@ -28,6 +29,57 @@ Depending on if you're using `npm` or `yarn`, run `npm install` or `yarn` in the
 Running your tests or deploying your application might fail at this point in time if you need to adapt to any breaking changes.
 We recommend updating your applications in one commit or pull request and making sure everything still works using your existing test suite.
 
+### Check for removed deprecated functions and replace them if required
+
+While the SAP Cloud SDK maintains backwards compatibility within a major version where possible, a new major release breaks compatibility where required to simplify the programming interface.
+Most of the removed functions had been deprecated before, so ideally they are not used anymore.
+The following sub-sections describe affected modules, functions and interfaces with instructions on how to replace them.
+
+#### HTTP Client
+
+The overload, that accepted `HttpRequestConfigWithOrigin` as a parameter, is removed and replaced by the function `executeHttpRequestWithOrigin`.
+
+#### Logger
+
+The field `logger` on the interface `LoggerOptions` was not used and is removed from the interface.
+
+#### Array Utils
+
+The function `variadicArgumentToArray` is replaced by the function `transformVariadicArgumentToArray`.
+
+#### JWT
+
+The generic types of `JwtKeyMapping` is simplified so the second type argument `JwtKeysT` are always strings.
```

Comment:
[req] The function name is listed, which looks good.
However, the section title is called "JWT", I as a user would not know whether this is related to a package that I'm using.
Instead of using "JWT", I would prefer using the npm package name. (@sap-cloud-sdk/connectivity?)

Reply:
Hi @jjtang1985 

thanks for your feedback, I think having npm package names as the first structure element makes sense. For the odata specific things I'm not sure what the second structure element should be, but I feel like we need a structure there because the text is quite long. I'm happy to take any suggestion you might have to improve it.

Please have another look.